core/src/main/scala/cats/collections/Predicate.scala

```diff
@@ -76,6 +160,15 @@ trait PredicateInstances {
     override def combine(l: Predicate[A], r: Predicate[A]): Predicate[A] = l union r
   }
 
+  implicit def predicateBool[A]: Bool[Predicate[A]] = new Bool[Predicate[A]] {
```

Comment:
we usually give longer names in cats to avoid the name aliasing problem with implicits.

e.g. `implicit def catsCollectionsPredicateBool: Bool[Predicate[A]] = ...`